metalog.js

```diff
@@ -287,6 +287,13 @@ class Logger extends events.EventEmitter {
   bind(application) {
     return new ApplicationLogger(this, application);
   }
+
+  // Checks whether passed `type` of logs is active.
+  //   type <string> logger type
+  // Returns: <boolean>
+  isActive(type) {
+    return this.active && !!(this.toStdout[type] || this.toFile[type]);
```

Comment:
I'd prefer a less cryptic way of expressing that:
```suggestion
    return this.active && Boolean(this.toStdout[type] || this.toFile[type]);
```